browser/src/UI/Actions.ts

```diff
@@ -34,7 +34,7 @@ export interface IBufferUpdateAction {
     type: "BUFFER_UPDATE",
     payload: {
         id: number,
-        version: number,
+        modified: number,
```

Comment:
Yeah I was going to do that, but I left it as is because I know some vimmers are hard core numbers because c, but yeah I was thinking about changing it to bool too, so will do :).